dorado/parallel_routing.py

```diff
@@ -12,21 +12,21 @@
 import os
 import re
 import string
-from .particle_track import Particle
+from .particle_track import Particles
 from multiprocessing import Pool
 
 
 # convert run script into a function that returns a single dictionary 'data'
 def run_iter(params):
```

Comment:
Looks like the parallel functions are still expecting attributes of `modelParams` that have been moved, I assume we want this to be using a `Particles` object instead?

Reply:
Yes that's probably true